### PR TITLE
Problem: (CRO-455) Byzantine validators are not slashed

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Add byzantine validators to slashing queue and slash them when wait time passes

### CODE REVIEW

File: chain-abci/src/app/mod.rs [hunk 1]
```diff
@@ -156,16 +158,29 @@ impl<T: EnclaveProxy> abci::Application for ChainNodeApp<T> {
...
-                    .unwrap()
+                let account_address = last_state
                     .validator_liveness
                     .get(&validator_address)
                     .expect("Validator not found in liveness tracker")
                     .address();
 
                 accounts_to_jail.push(account_address);
+
+                let slashing_ratio = last_state.slashing_config.byzantine_slash_percent;
```
[the remark this answers is not in the document]
Comment (devashishdxt) [reply chain-abci/src/app/mod.rs]: This, in my view, is not difficult to modify later. We can modify the implementation once we have all the scenarios sketched out in `chain-docs`.

**Comment by tomtau** [comment]
@devashishdxt init config is currently feature-guarded, as it's not needed in tx-validation enclave etc., so compiling enclaves fails due to this:
```
error[E0432]: unresolved import `crate::init::config`
614	 --> /tmp/drone-6NS9G1rMvn8mGTkF/drone/src/chain-core/src/init/coin.rs:6:18
615	  |
616	6 | use crate::init::config::SlashRatio;
617	  |                  ^^^^^^ could not find `config` in `init`
618	
619	
620	error[E0432]: unresolved import `crate::init::config`
621	 --> /tmp/drone-6NS9G1rMvn8mGTkF/drone/src/chain-core/src/state/account.rs:4:18
622	  |
623	4 | use crate::init::config::SlashRatio;
624	  |                  ^^^^^^ could not find `config` in `init`
625	
626	
627	error: aborting due to 2 previous errors
628	
629	For more information about this error, try `rustc --explain E0432`.
630	error:
```

`SlashRatio` could either be moved out to a separate module, or its uses in Coin and StakedState can be feature guarded (if it can be done -- for StakedState, it may not)

File: chain-abci/src/app/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomtau) [reply chain-abci/src/app/mod.rs]: ok

**Pull request merged**